documentation/rfc/autocomplete.md

```diff
@@ -55,6 +55,6 @@ and finally by the quality.
 If a request returns a lot of admins, the response will be filled with the ten most important admins 
 and won't have any stop_area or way.
 
-Most of the times the complain we got on the autocompletion is that a stop_area can't be found.
+Most of the times the complains we got on the autocompletion is that a stop_area can't be found.
```

Comment:
Either: 'Most of the time~s~' (because uncountable) or simply `Most of the complains we've got are [...]` (you need to use present perfect as there is no time reference)